package/plugin-gargoyle-qr-code/files/www/hooks/login/090-qr_code.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * This program is copyright © 2020 Rouven Spreckels and is distributed under the terms of the GNU GPL
+ * version 2.0 with a special clarification/exception that permits adapting the program to
+ * configure proprietary "back end" software provided that all modifications to the web interface
+ * itself remain covered by the GPL.
+ * See http://gargoyle-router.com/faq.html#qfoss for more information
+ */
+
+// ECMAScript 6 (ES6) Strict Mode Code
+"use strict";
+
+addLoadFunction(() =>
+{
+	// Whether shared login hook resources of plugin are loaded.
+	const pkg = uciOriginal.get("gargoyle", "qr_code", "pkg");
```

Comment:
Use of `const` where `var` is likely sufficient in many locations. Save those bytes 2 at a time!
Again I won't say no, but consider their use.

Reply:
It seems `const`s survive terser and there is no switch to get rid of them. I would prefer to go with `let` then because of block scope and no hoisting.